phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/engines/EnginesController.java

```diff
@@ -64,7 +65,7 @@ public EnginesController(ViewEngines viewEngines, RepositoryManager repositoryMa
         this.themeManager = themeManager;
 
         this.repositoryManager.addCallbacks(this::populateView,
-                e -> Platform.runLater(() -> viewEngines.showFailure()));
+                e -> Platform.runLater(() -> viewEngines.showFailure(tr("Loading engines failed."), Optional.of(e))));
```

Comment:
Yes, I'm currently using `Optional.of(e)` everywhere. However, I wanted to have the possibility to call it with `Optional.empty()`. It might happen that you want to show a `FailurePanel` but do not have an exception.